### DISCUSSION TITLE
What is the compatibility of Element-Plus？

### DISCUSSION THREAD

**Discussion by wdlnydhcg** [element-plus/element-plus#3469] [Q&A] [answered]
I kown vue3 compatibility firefox35 and i can normal run it,but i can't use element-plus,There are no errors on the console, Although I can see this input box, I can't actually enter anything.

**Comment by sxzz** [comment, marked as answer]
Element Plus can run on browsers that supports ES2018 and ResizeObserver. If you really need to support outdated browsers, please add Babel and Polyfill yourself.

**Reply by wdlnydhcg** [reply]
i kown should use babel and polyfill,  Actually，i've already done that。But the console has no error log，UI components can be displayed, but cannot be used。

**Reply by sxzz** [reply]
Please provide minimum reproduction link.

**Comment by wdlnydhcg** [comment]
This is the project address(https://github.com/wdlnydhcg/vue3-ele-compatibility)，I can access it on chrome90, but I can't use the UI components on firefox35.



this is firefox35&nbsp;




this is chrome90









------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "element-plus/element-plus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月18日(星期六) 上午10:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [element-plus/element-plus] What is the compatibility of Element-Plus？ (Discussion #3469)





 
Please provide minimum reproduction link.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by sxzz** [reply]
The image does not seem to be uploaded successfully.

**Reply by sxzz** [reply]
Firefox 35 is broken in my Mac machine. So I can't test it.

**Comment by wdlnydhcg** [comment]
Do you need an installation file for firefox？
Maybe you can get it here&nbsp; （http://ftp.mozilla.org/pub/firefox/releases/35.0.1/mac/）




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "element-plus/element-plus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月18日(星期六) 晚上9:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [element-plus/element-plus] What is the compatibility of Element-Plus？ (Discussion #3469)





 
Firefox 35 is broken in my Mac machine. So I can't test it.
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by sxzz** [reply]
I tried, but the application can't work. Could you please try newer version.

**Reply by sxzz** [reply]
It seems that Firefox 35 can't run on macOS Big Sur.
